.travis.yml

```diff
@@ -20,7 +20,7 @@ install:
     - which javac && javac -version
     - which java && java -version
     #Copy protobuf binaries
-    - wget https://github.com/Intel-HLS/GenomicsDB/releases/download/0.4.0/protobuf-3.0.2-trusty.tar.gz -O protobuf-3.0.2-trusty.tar.gz
+    - wget https://github.com/Intel-HLS/GenomicsDB/releases/download/0.5.0/protobuf-3.0.2-trusty.tar.gz -O protobuf-3.0.2-trusty.tar.gz
```

Comment:
That's not right - the protobuf tar.gz is independent of the GenomicsDB release. The binaries were included as part of 0.4 release.